ros2cli/ros2cli/node/direct.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+
+# TODO(mikaelarguedas) revisit this once it's specified
+HIDDEN_NODE_PREFIX = '_'
+
+DEFAULT_TIMEOUT = 0.1
```

Comment:
The previous incarnation of the tool used 0.01s (see https://github.com/ros2/cli_tools/blob/de404fdf6fc18deedf247c2ab2cabff4f532731f/cli_tools_py/rosnode_list.py#L27). When I tested the command line tools I sometimes didn't get the complete result with that short timeout and therefore increased it. With 0.1s I got complete results for all of my tests. That certainly depends in the network topology and size of the graph. If you think we should use a different default value please propose one.